service/src/geopm_pio.h

```diff
@@ -100,6 +100,11 @@ int geopm_pio_control_description(const char *control_name,
                                   size_t description_max,
                                   char *description);
 
+int geopm_pio_signal_info(const char *signal_name,
```

Comment:
Docu?

I see by inspecting the implementation that everything past the first argument is supposed to be output.  If we're moving away from having all these APIs described in the ronn files, maybe we should put some notes here?